vue.config.js

```diff
@@ -6,5 +6,11 @@ module.exports = defineConfig({
 		vuetify: {
 			// https://github.com/vuetifyjs/vuetify-loader/tree/next/packages/vuetify-loader
 		}
+	},
+	devServer: {
```

Comment:
This is necessary for your locally running devServer to be able to use web workers with shared data arrays. It requires https and same origin normally, so I think it should work in production but we may need to set some cors thing for that as well.